wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+ConversationContentViewControllerDelegate.swift

```diff
@@ -176,6 +176,32 @@ extension ConversationViewController: ConversationContentViewControllerDelegate
         }
     }
 
+    func conversationContentViewController(
+        _ controller: ConversationContentViewController,
+        didDeleteMultipartMessage message: any ZMConversationMessage,
+        withAttachments attachments: [MultipartMessageData.Attachment],
+        deletionType: DeletionType
+    ) {
+        switch deletionType {
+        case .everywhere:
+            Task {
+                let deleteNodesUseCase = wireCellsFactory.makeDeleteNodesUseCase()
+                do {
+                    try await deleteNodesUseCase.invoke(nodeIDs: attachments.map(\.nodeID))
+                    WireLogger.conversation.info("Deleted files for message: \(message.nonce?.uuidString ?? "nil")")
```

Comment:
suggestion: use log attributes
```suggestion
                    WireLogger.conversation.info("Deleted files for message, attributes: .nonce: message.nonce?.uuidString)
```

Reply:
done